perfmetrics/scripts/run_e2e_tests.sh

```diff
@@ -260,14 +262,14 @@ function main(){
   run_e2e_tests_for_flat_bucket &
   e2e_tests_flat_bucket_pid=$!
 
-  set -e
-
   wait $e2e_tests_flat_bucket_pid
   e2e_tests_flat_bucket_status=$?
 
   wait $e2e_tests_hns_bucket_pid
   e2e_tests_hns_bucket_status=$?
 
+  set -e
+
   print_test_logs
 
   if [ $e2e_tests_flat_bucket_status != 0 ];
```

Comment:
the tests can fail on both flat and hns buckets also, so can we handle that case i.e. if both exit codes are non zero then print both flat and hns failed

Reply:
Good point. Added thanks.